src/layout/flex-grid/src/index.tsx

```diff
@@ -83,28 +83,32 @@ export const Row: React.FC<RowProps> = ({
       }}
       {...props}
     >
-      {childrenArray.map((child: any) => React.cloneElement(child, { cols }))}
+      {childrenArray.map((child: React.ReactNode) =>
+        React.isValidElement(child)
+          ? React.cloneElement(child, { cols })
+          : child
+      )}
     </div>
   )
 }
 
 interface ColProps {
-  children: any
-  cols?: number
-  className?: string
-  span?: any
-  sx?: any
-  offset?: any
+  cols?: number | number[]
+  span?: number | number[]
+  offset?: number | number[]
 }
 
 export const Col: React.FC<ColProps> = ({
   children,
   cols = 12,
   span,
   offset,
-  sx = {},
   ...props
 }) => {
+  if (Array.isArray(span) && !Array.isArray(cols)) {
+    cols = new Array(span.length).fill(cols)
```

Comment:
Fill will blow up in IE, and node modules don't get transpiled in our apps, so I'm assuming this will be an issue (unless typscript does something?)